ui/app/components/app/modals/new-account-modal/new-account-modal.component.js

```diff
@@ -35,6 +35,14 @@ export default class NewAccountModal extends Component {
     }
   }
 
+  onKeyDown = (e) => {
+    const { t } = this.context
+
+    if (e.keyCode === 39 && this.state.alias === '') {
+      this.setState({ alias: t('newAccountNumberName', [this.props.newAccountNumber]) })
```

Comment:
I thought this was a common pattern... I only added it because I kept trying to do it when interacting with this modal... maybe I'm thinking of some similar interaction... I'll have to investigate

Reply:
It sounds like something you'd expect with an autocomplete suggestion - not with a form.

[The MDN article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefplaceholder) seems to strongly recommend against this sort of thing as well, and against using `placeholder` in general. I think we can avoid it.

Reply:
Okay, 99d4ea469 changes it so it works like "the field was automatically populated with "Account N", and we'd just leave it up to the user to change it if they want to"